### PR TITLE
Update headers to 2020

### PR DESCRIPTION
**Pull request by ruspl-afed**
Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#13](https://github.com/arsysop/mola/issues/13) is assigned to @eparovyshnaya[/z](https://www.0crat.com/u/eparovyshnaya), here is [why](https://www.0crat.com/footprint/CEN8A36TW/9c2f9630-d651-438d-820a-f6b76dd075c1); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @ruspl-afed[/z](https://www.0crat.com/u/ruspl-afed) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2018 ArSysOp
+# Copyright (c) 2018-2020 ArSysOp
```
Comment (eparovyshnaya) [inline .gitignore]: Shouldn't we use comma instead of hyphen here?

**Review by eparovyshnaya — CHANGES_REQUESTED** [review]
There are 8 similar issues, one per file: according to [Eclipse Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) we should use `{yearOfInception}, {yearOfLastUpdate}` format in the *copyright* statement - with comma symbol instead of hyphen.

File: .gitignore (hunk 1 above)
Comment (ruspl-afed) [reply .gitignore]: I don't think so because currently we are using Apache 2.0

Comment (eparovyshnaya) [reply .gitignore]: Handbook declares *copyright* statement stracture without regard to any particular license. As the project is Eclispse-oriented, I think we should follow these rules.

Comment (ruspl-afed) [reply .gitignore]: agreed, please review the updated version

**Review by eparovyshnaya — APPROVED** [review]

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @eparovyshnaya[/z](https://www.0crat.com/u/eparovyshnaya)[/z](https://www.0crat.com/u/eparovyshnaya) didn't make CR comments

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @eparovyshnaya[/z](https://www.0crat.com/u/eparovyshnaya)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @ruspl-afed[/z](https://www.0crat.com/u/ruspl-afed)

**Pull request merged**